earth_enterprise/rpms/build.gradle

```diff
@@ -269,6 +271,8 @@ task openGeeCommonRpm(type: GeeRpm, dependsOn: openGeeSharedFiles) {
     requires('chkconfig')
     requires('shadow-utils')
     requires('sudo')
+    conflicts('opengee-postgis', '2.0', LESS )
```

Comment:
Q: Did you have to add the `opengee-postgis` `conflicts` headers to `opengee-common` for some reason?

Reply:
I'm not the one that added the line, so I'm just guessing, but I think the conflict makes sense because common includes postgres which will only work with certain versions of postgis.

Reply:
`common` is separate from `postgres`. In fact `postgres` requires `common`.